docs/herramienta_tests.md

```diff
@@ -1,4 +1,4 @@
-# Requisitos para la elección de la herramienta
+# Requisitos para la elección de la herramienta para testear
```

Comment:
¿De veras?

Reply:
Habiendo leído de nuevo la documentación creo que afina nombrarlo como hago en [e1d620c](https://github.com/antoniorr02/MenuConsulter/pull/22/commits/e1d620c85a49a45fd46cacf002d79654910c5fba)

Reply:
¿Y has leído también [este comentario](https://github.com/antoniorr02/MenuConsulter/pull/22#discussion_r1856027602) que te puse tres o cuatro revisiones más atrás?
Si no os leéis o no tenéis presentes ni siquiera los comentarios anteriores, y volvéis a poner lo mismo, de veras que no sé si merece la pena emplear tiempo en deciros cuál es es el error y donde está. Entramos otra vez en el bucle: te digo "Está mal" y ya está.

Reply:
Si lo leí, pensaba que lo solucione en el commit que cerró el respectivo [comentario](https://github.com/antoniorr02/MenuConsulter/pull/22#discussion_r1857117969). 

Y hasta ahora no ha vuelto a salir el error porque pensaba que ya lo había corregido, si no es así como se supone que debería hacer procedido, ¿qué he hecho mal respecto a la metología?

Reply:
¿Te parece que volver a poner algo que ya se ha dicho que estaba mal es "seguir la metodología"?

Reply:
Me dijo que era incorrecto poner "herramienta de test", con lo que lo cambie a "herramienta" de manera quizá instintiva. Ahora que me he leido de nuevo por cuarta o quinta vez el guión lo he nombrado como "testing framework" para que quede claro que la herramienta puede tener aserciones, test runner, etc. En tal caso y en este punto, ¿a dónde estamos intentando llegar en este comentario?

Reply:
A lo que ya te he dicho. Está mal. Léete el guión.

Reply:
En el guión pone textualmente lo que le acabo de decir y lo que indico en mi archivo:

"En general, se habla de testing framework cuando incluye dos o tres de estas características. Conviene en todo caso consultar el panorama (y las mejores prácticas) en el lenguaje de nuestra elección para aclarar qué herramientas existen, qué funcionalidad precisa tienen, y finalmente elegir lo más conveniente.

Tal como se ha hecho en el [objetivo anterior](https://jj.github.io/IV/documentos/proyecto/3.Automatizar), hay que llevar a cabo una elección documentada de las herramientas que se van a usar."

No acabo de entender entonces el problema de titular a los requisitos de cualquiera de las formas que me ha rechazado por ahora.

Reply:
Podría explicarme en tanto, ¿por qué está mal?

Reply:
Resuelto en [5f80c8d](https://github.com/antoniorr02/MenuConsulter/pull/22/commits/5f80c8d60ad66c60fbc411ad3481da296a1e04ad)